pkg/build/builder/dockerutil.go

```diff
@@ -2,11 +2,21 @@ package builder
 
 import (
 	"os"
+	"time"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/golang/glog"
 	"github.com/openshift/source-to-image/pkg/tar"
 )
 
+var (
+	// Number of retries of pushing the built Docker image into configured
+	// repository
+	DefaultPushRetryCount = 2
```

Comment:
I think waiting 60 seconds is too much, I would rather increase the number of retries to 3-4